repository/src/main/java/com/iluwatar/repository/App.java

```diff
@@ -84,17 +85,17 @@ public static void main(String[] args) {
     nasta.setSurname("Spotakova");
     repository.save(nasta);
 
-    LOGGER.info("Find by id 2: {}", repository.findOne(2L));
+    LOGGER.info("Find by id 2: {}", repository.findById(2L));
```

Comment:
`findById` returns an `Optional`, probably we don't want to log that but its contents